Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/MetaAdapter.h

```diff
@@ -27,7 +27,7 @@ namespace AZ::DocumentPropertyEditor
         virtual void HandleReset() = 0;
         virtual void HandleDomChange(const Dom::Patch& patch) = 0;
 
-        static bool IsRow(const Dom::Value& domValue);
+        using DocumentAdapter::IsRow;
```

Comment:
question: is this needed since the 'IsRow' function is already a public method in the base class of 'DocumentAdapter' ?

Reply:
yes, it's needed. Without this, the other overload of IsRow masks the original instance:
Dom::Path GetRowPath(const Dom::Path& sourcePath) const